src/Controls/src/Core/Compatibility/Handlers/NavigationPage/iOS/NavigationRenderer.cs

```diff
@@ -111,8 +111,7 @@ public override void DidRotate(UIInterfaceOrientation fromInterfaceOrientation)
 
 			View.SetNeedsLayout();
 
-			var parentingViewController = GetParentingViewController();
-			parentingViewController?.UpdateLeftBarButtonItem();
+			UpdateFlyoutMenuButton();
```

Comment:
.NET MAUI supports iOS 12.2 or higher is required, and this  is not called in 9.0+. Should not be necessary.